### ISSUE TITLE
Using multiple default images to make the array of project images on the homepage more colourful

### ISSUE THREAD

**Issue by avicaplan** [awesomefoundation/awesomebits#366]
Changing up the default grey image for projects for which no images were uploaded to randomly select different coloured versions of the same image or one of a series of more vibrant illustrations on the same theme would make the homepage more lively when there are a bunch of projects without images showing.

Alternately, we could consider excluding from the homepage projects without images.

**Comment by jcn** [comment]
I would be curious whether all deans know that they can add images after the fact. I have been thinking that marking a project as a winner should not just be a toggle, but an entirely separate form which includes the ability to edit the photos. Before we make significant changes to the process, I would want to talk to our users first to make sure they know what's actually going on and the options they have.

(also, personally I think that a multi-colored grid might look a little funny, but I could be convinced otherwise).

**Comment by avicaplan** [comment]
Yes, I think such a workflow would make sense! And agreed that multi-coloured might not really be the best way to go. Ideal might be to have someone with some design skills develop a collection of default images that work with the site's design but are more interesting visually than our current default.
